include/flamegpu/runtime/environment/DeviceEnvironmentDirectedGraph.cuh

```diff
@@ -527,6 +528,17 @@ __device__ __forceinline__ id_t DeviceEnvironmentDirectedGraph::getVertexID(cons
 #endif
     return getVertexProperty<id_t>("_id", vertex_index);
 }
+
+__device__ __forceinline__ unsigned int DeviceEnvironmentDirectedGraph::getVertexIndex(id_t vertex_id) const {
+#if !defined(FLAMEGPU_SEATBELTS) || FLAMEGPU_SEATBELTS
+    const unsigned int VERTEX_COUNT = detail::curve::DeviceCurve::getVariableCount("_index_map", graph_hash ^ detail::curve::Curve::variableHash("_environment_directed_graph_vertex"));
```

Comment:
As discussed in meeting, this limit will nolonger be enforced. Instead IDs will have a separate max.